test/test.js

```diff
@@ -1,29 +1,33 @@
 // Packages
-const test = require('ava')
+const test = require('ava');
```

Comment:
Where are all these formatting changes coming from? Did you forget to add a devDependency such as prettier?

Reply:
I ran Prettier, yes. We can add that here, or a separate PR?

Reply:
Added here 😄